### PR TITLE
Small addition to the Python doc for Outside EU codes

### PR DESCRIPTION
**Pull request by teolemon**
Small addition to the Python doc for Outside EU codes
Related: https://github.com/openfoodfacts/openfoodfacts-server/issues/4030

### CODE REVIEW

File: scripts/packager-codes/non-eu/README.md [hunk 1]
```diff
@@ -32,3 +32,9 @@ To display the status of the locally downloaded files as compared to the remote:
 ````shell script
 python packager_codes.py status packager_codes_data
 ````
+
+To extract codes
+````
+find . -name "*.pdf"
+python ./pdf_extraction.py
```
Comment (hangy) [inline scripts/packager-codes/non-eu/README.md:39]: This doesn't do anything with the file list, does it? Does the output need to be piped into python or used as the argument list?

Also, should this note that all of this syntax requires *NIX or at least GNU findutils, or any other specific platform requirements?

Comment (teolemon) [reply scripts/packager-codes/non-eu/README.md:39]: I'm working on it tonight. Marking as WIP

Comment (hangy) [reply scripts/packager-codes/non-eu/README.md:39]: oh ok. this confused me a bit. 😀 

Are you editing this in the GitHub website or do you push it manually?

Comment (teolemon) [reply scripts/packager-codes/non-eu/README.md:39]: For this PR, from the GitHub web interface. On multifile PRs: commandline

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**